packages/repository/src/repositories/relation.repository.ts

```diff
@@ -119,11 +125,17 @@ export class DefaultBelongsToEntityCrudRepository<
   TargetId,
   TargetRepository extends EntityCrudRepository<TargetEntity, TargetId>
 > implements BelongsToRepository<TargetEntity> {
+  /**
+   * Constructor of DefaultBelongsToEntityCrudRepository
+   * @param getTargetRepository the related target model repository instance
```

Comment:
Ditto. _the related target model repository instance_ is misleading, please fix the comment to mention a getter.